tests/default_test_settings.py

```diff
@@ -15,7 +11,7 @@
         index_config={
             "DEFAULT_PREFIX": "testprefix:",
             "PREPEND_PREFIX": True,
-            "ADD_PREFIX_ALIAS": True,
+            "ADD_PREFIX_ALIAS": False,

```

Comment:
i think [this test](https://github.com/uc-cdis/indexd/blob/f2557f5/tests/test_client.py#L1885-L1902) is not testing the `ADD_PREFIX_ALIAS` feature anymore, it's probably just passing because `PREPEND_PREFIX` is True... but we need to test this feature because it's used in prod

maybe that's not in the scope of this PR

Reply:
oh good point, I'll try to fix this